ONSdigital/dp-frontend-router
# Release 1.26.0

### What

Performance improvements
- Less calls to Zebedee (no app retries, these are handled on the platform)
- No longer calling random files from Zebedee

### How to review

- Check that develop is working as expected
- Check that nothing has slipped into this release that shouldn't be in it

### Who can review

Describe who worked on the changes, so that other people can review.
